accord-core/src/main/java/accord/primitives/Range.java

```diff
@@ -216,12 +216,12 @@ private Range(RoutingKey start, RoutingKey end)
         this.end = end;
     }
 
-    public final RoutingKey start()
+    public RoutingKey start()
```

Comment:
in `TokenRange` we kept doing casts making the code ugly... this change was to try to make that logic cleaner... its 100% style change